evergreen.yml

```diff
@@ -395,12 +395,12 @@ functions:
   f_dry_run_workloads:
   - command: subprocess.exec
     params:
-      working_dir: .
+      working_dir: src
       binary: /bin/bash
       args:
-      - ${workdir}/src/scripts/dry-run-workloads.sh
+      - scripts/dry-run-workloads.sh
       env:
-        GENNY: ${workdir}/dist/bin/genny
+        GENNY: scripts/genny
```

Comment:
hm maybe `dry-run-workloads.sh` should know about `scripts/genny` now?

Reply:
It does and uses that in the default case! Figured I'd keep setting the variable to be explicit, let me know if you disagree.